starter-code/movies/routes/celebrities.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const Celeb = require('../models/celebrities');
+
+const router = express.Router();
+
+router.get('/celebs', (req, res, next) => {
+  Celeb.find({}).sort().exec((err, celebs) => {
+    if (err) return next(err)
+    res.render('celebs/index', { celebs })
+  });
+});
+
+router.get('/celebs/:id/show', (req, res, next) => {
+  const celebs = {
+    celebID: req.params.id
+  }
+  Celeb.findById(celebs.celebID, (err, celeb) => {
+    if (err) return next(err)
+     res.render('celebs/show', { celeb })
+   });
+});
+
+router.get('/celebs/new', (req, res, next) => {
+  res.render('celebs/new')
+});
+
+router.post('/celebs', (req, res, next) => {
+  const celeb = new Celeb ({
+  	name: req.body.name,
+  	occupation: req.body.occupation,
+  	catchPhrase: req.body.catchPhrase,
+  })
+  celeb.save((err,celeb) => {
+  	if (err) res.redirect('celebs/new')
+  res.redirect('celebs')
+  })
+})
+
+router.post('/celebs/:id/delete', (req,res,next) => {
```

Comment:
Just know that there is also a "DELETE" http verb that you can use semantically, and then the route could just be /celebs/:id with a DELETE instead of POST :)

Reply:
I tried it at first, never managed to make it work, but it was probably due to another unrelated bug that I solved later, will try again